lib/mongo/uri.rb

```diff
@@ -455,8 +455,7 @@ def self.uri_option(uri_key, name, extra = {})
     uri_option 'tlsallowinvalidcertificates', :ssl_verify, :type => :ssl_verify
     uri_option 'tlscafile', :ssl_ca_cert
     uri_option 'tlscertificatekeyfile', :ssl_cert
-    uri_option 'tlsclientkeyfile', :ssl_key
-    uri_option 'tlscertificatekeypassword', :ssl_key_pass_phrase
+    uri_option 'tlscertificatekeyfilepassword', :ssl_key_pass_phrase
```

Comment:
Per https://jira.mongodb.org/browse/RUBY-1676 I'm thinking this change should also have an accompanying documentation change, no?

Reply:
Done